src/Arcus.WebApi.Security/Authentication/X509Validation.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Arcus.WebApi.Security.Authentication 
+{
+    /// <summary>
+    /// Represents which value of the client <see cref="X509Certificate2"/> should be validated in the <see cref="CertificateAuthenticationFilter"/>.
+    /// </summary>
+    public enum X509Validation
```

Comment:
This is interesting - This enum is not really validating the cert but more defining the requirements, no?

How about this:
```suggestion
    public enum X509ValidationRequirement
```

Reply:
Yes, it is indeed more a requirement. Good suggestion!